manifests/k/KINGSGROUP/KOA/1.0/KINGSGROUP.KOA.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: KINGSGROUP.KOA
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: KINGS GROUP

```

```suggestion
Publisher: FunPlus, Inc.
```
This should match Control Panel